centreon/packages/ui/src/Listing/Cell/DataCell.tsx

```diff
@@ -103,7 +104,10 @@ const DataCell = ({
       return (
         <Cell
           isRowHighlighted={isRowHighlighted}
-          style={{ gridColumn }}
+          style={{
+            gridColumn,
+            paddingLeft: theme.spacing(1.5)
+          }}
```

Comment:
Is it possible to use a className instead?

Reply:
not sure! it's so difficult to work with mui nested classes architecture